src/lib/components/UploadBtn.svelte

```diff
@@ -1,23 +1,72 @@
 <script lang="ts">
+	import { PdfUploadStatus } from "$lib/types/PdfChat";
+	import { createEventDispatcher, onDestroy } from "svelte";
 	import CarbonUpload from "~icons/carbon/upload";
 
 	export let classNames = "";
+	export let multimodal = false;
 	export let files: File[];
-	let filelist: FileList;
+	export let uploadPdfStatus: PdfUploadStatus;
 
```

Comment:
```suggestion
	export let uploadPdfStatus: PdfUploadStatus | undefined = undefined;
```

fix the missing uploadPdfStatus in type checks